index.js

```diff
@@ -47,10 +47,20 @@ class ServerlessCustomDomain {
   initializeVariables() {
     if (!this.initialized) {
       // Sets the credentials for AWS resources.
-      const awsCreds = this.serverless.providers.aws.getCredentials();
+      let awsCreds = this.serverless.providers.aws.getCredentials();
+      let awsRegion = this.serverless.providers.aws.getRegion();
       AWS.config.update(awsCreds);
+      const domainProfile = this.serverless.service.custom.customDomain.route53Profile;
+      if (domainProfile) {
+        awsCreds = new AWS.SharedIniFileCredentials({ profile: domainProfile });
+        awsRegion = this.serverless.service.custom.customDomain.route53Region || 'us-east-1';
```

Comment:
Why `us-east-1`. Would it make more sense to default to `awsRegion` instead?

Reply:
@aoskotsky-amplify Good point, updated.